src/cli/serve/integration_tests/reload_logging_config.test.js

```diff
@@ -21,34 +21,30 @@ import { spawn } from 'child_process';
 import fs from 'fs';
 import path from 'path';
 import os from 'os';
-import mkdirp from 'mkdirp';
```

Comment:
__question:__ I guess it's (and few places below where these packages are used) accidentally removed?